src/borg/testsuite/chunker.py

```diff
@@ -82,11 +83,10 @@ def twist(size):
                         for seed in (1849058162, 1234567653):
                             fh = BytesIO(data)
                             chunker = Chunker(seed, minexp, maxexp, maskbits, winsize)
-                            runhash = blake2b()
-                            for c in chunker.chunkify(fh, -1):
-                                runhash.update(blake2b(c).digest())
-                            overall_hash.update(runhash.digest())
+                            chunks = [blake2b_256(b'', c) for c in chunker.chunkify(fh, -1)]
+                            runs.append(blake2b_256(b'', b''.join(chunks)))
 
         # The "correct" hash below matches the existing chunker behavior.
```

Comment:
Just to make sure: the known-good hash was computed by running the test using the chunker code from *before* your change?

Reply:
I've lost track, but I just ran the test against the current master branch and it passed with the same good-hash, so it's good.

Reply:
ok, great!